server/urls.py

```diff
@@ -4,7 +4,7 @@
 
 urlpatterns = [
     path('', views.index, name="index"),
-    path('volunteer_table', views.show_all_volunteers, name='show_all_volunteers'),
+    path('volunteer_table/', views.show_all_volunteers, name='show_all_volunteers'),
```

Comment:
Add a trailing slash to help_table too

Reply:
just did. can you check if this is what you meant?